.github/workflows/release.yml

```diff
@@ -31,5 +31,5 @@ jobs:
           # This expects you to have a script called release which does a build for your packages and calls changeset publish
           publish: npm run release
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.PUBLISH_GITHUB_TOKEN }}
```

Comment:
Oops, we need to use a PAT instead of the GH actions token so our workflows can open PRs. A recent GitHub security fix restricted GH actions from being able to open PRs by default.